packages/analytics-js-integrations/src/integrations/Ninetailed/browser.js

```diff
@@ -51,14 +52,16 @@ class Ninetailed {
     window.ninetailed.track(event, properties);
   }
   page(rudderElement) {
-    const { message } = rudderElement;
-    const { properties } = message;
-    if (properties) {
-      properties.url = window.location.href;
-      window.ninetailed.page(properties);
-      return;
+    if (this.sendPageEvents) {
+      const { message } = rudderElement;
+      const { properties } = message;
+      if (properties) {
+        properties.url = window.location.href;
+        window.ninetailed.page(properties);
+        return;
+      }
+      window.ninetailed.page();
```

Comment:
Is `properties.url` not required here then ?

Reply:
Properties is an optional parameter, if not sent ninetailed itself captures the details.
Yes `properties.url` is not required

Reply:
In case of `properties` present in payload & we set them `window.ninetailed.page(properties)`, does ninetailed appends it's auto-captured properties ?

Reply:
Will need to check on that

Reply:
@krishna2020 Yes I just checked and ninetailed appends the auto-captured properties